students/parkdaehyun/users/views.py

```diff
@@ -34,5 +32,15 @@ def post(self, request):
         )
 
         return JsonResponse({"message" : "SUCCESS"}, status = 201)
-        
-        
+
+class Login(View):
```

Comment:
View class에는 LoginView와 같이 상속받은 View를 명시해주는 것이 컨벤션입니다~
```suggestion
class LoginView(View):
```

Reply:
수정했습니다~ 꼼꼼한 리뷰 감사합니다!